activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add `SKIP_TEST_DATABASE` environment variable to disabling modifying the test database when `rails db:create` and `rails db:drop` are called.
```

Comment:
```suggestion
*   Add `SKIP_TEST_DATABASE` environment variable to disable modifying the test database when `rails db:create` and `rails db:drop` are called.
```

Either that or `... to disable modification of the test` or `...variable for disabling modification of the test`

`to disabling modifying the test` does not sounds right.